inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -834,7 +833,8 @@ class MKLDNNNode {
     }
 
 private:
-    void redefineOutputMemory(const std::vector<std::vector<size_t>> &newShapes);
+    std::unique_ptr<MemoryDesc> getInputMemDescAtPort(size_t portNum) const;
+    std::unique_ptr<MemoryDesc> getOutputMemDescAtPort(size_t portNum) const;
```

Comment:
Please change the method names otherwise people will get weird errors when they forget specify a template parameter for similarly named template methods.